src/components/CustomerQuota/CheckoutSuccess/CheckoutSuccessCard.tsx

```diff
@@ -210,7 +211,7 @@ export const CheckoutSuccessCard: FunctionComponent<CheckoutSuccessCard> = ({
               color={color("blue-green", 40)}
               style={sharedStyles.icon}
             />
-            <AppText style={sharedStyles.statusTitleWrapper}>
+            <View style={sharedStyles.statusTitleWrapper}>
```

Comment:
I changed it to View because when use AppText, got errors and cannot render in element inspector. It produces the same screen. 

If not in future, cannot inspect. so still revert change?